backend/api-service/src/backend_firestore.go

```diff
@@ -63,6 +63,10 @@ func (fb *FirestoreBackend) DeleteLocation(ctx context.Context, id string) error
 	return fb.deleteDoc(ctx, "locations", id)
 }
 
+func (fb *FirestoreBackend) DeleteAlert(ctx context.Context, id string) error {
+	return fb.deleteDoc(ctx, "alerts", id)
```

Comment:
Can this be pulled out to a `const` and reused?

Reply:
Yep! I pulled out the strings for the other objects too to be consistent.